test/tests/interfaces/pbs_sched_interface_test.py

```diff
@@ -0,0 +1,143 @@
+# coding: utf-8
+
+# Copyright (C) 1994-2017 Altair Engineering, Inc.
+# For more information, contact Altair at www.altair.com.
+#
+# This file is part of the PBS Professional ("PBS Pro") software.
+#
+# Open Source License Information:
+#
+# PBS Pro is free software. You can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free
+# Software Foundation, either version 3 of the License, or (at your option) any
+# later version.
+#
+# PBS Pro is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Commercial License Information:
+#
+# The PBS Pro software is licensed under the terms of the GNU Affero General
+# Public License agreement ("AGPL"), except where a separate commercial license
+# agreement for PBS Pro version 14 or later has been executed in writing with
+# Altair.
+#
+# Altair’s dual-license business model allows companies, individuals, and
+# organizations to create proprietary derivative works of PBS Pro and
+# distribute them - whether embedded or bundled with other software - under
+# a commercial license agreement.
+#
+# Use of Altair’s trademarks, including but not limited to "PBS™",
+# "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
+# trademark licensing policies.
+
+from tests.interfaces import *
+from ptl.lib.pbs_ifl_mock import MGR_CMD_CREATE
+
+
+@tags('multisched')
+class TestSchedulerInterface(TestInterfaces):
+    """
+    Test suite to test different scheduler interfaces
+    """
+
+    def setUp(self):
+        # delete all schedulers except default
+        self.server.manager(MGR_CMD_DELETE, SCHED, id="@default")
+        TestInterfaces.setUp(self)
+
+    def test_create_successful_scheduler_object(self):
+        """
+        Creates a scheduler object succcessfully .
+        """
+        a = {'sched_cycle_length': 12000}
+        rc = self.server.manager(MGR_CMD_CREATE, SCHED, a, id="test1")
+        self.assertTrue(rc == 0)
+
+    def test_list_schedulers(self):
+        """
+        List a scheduler object.
+        """
```

Comment:
so aren't we changing framework with this change? what does PTL give today if we do list sched/print sched? will it break any of the existing test cases where "print sched" is used?